Gems/EMotionFX/Code/Tests/ProvidesUI/AnimGraph/AnimGraphModelTests.cpp

```diff
@@ -138,6 +138,10 @@ namespace EMotionFX
             case QtWarningMsg:
             case QtCriticalMsg:
             case QtFatalMsg:
+                if (msg.contains("has active key-value observers (KVO)! These will stop working now that the window is recreated, and will result in exceptions when the observers are removed"))
```

Comment:
Isn't there is better way to filter it other than a string match? I mean qt could decide to tweak their comments and this filter won't work anymore.
At the least maybe just filter on "has active key-value observers (KVO)"?

Reply:
I think it is OK. Even if they were to change this error string, we very rarely upgrade to a new version of Qt (typically year+ in between).

Reply:
That's why I noted in the commit message that the warning has already been removed in the current Qt 5.15 and Qt 6 branches. It's already pretty much guaranteed to not change.